monitoring_demo/kruize/autotune.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
Please name this file as kruize.py, we dont want to highlight autotune here

Reply:
Done